### DISCUSSION TITLE
What version is the json if I want to use C++11 in project?

### DISCUSSION THREAD

**Discussion by AugF** [nlohmann/json#2991] [Q&A] [answered]
What version is the json if I want to use C++11 in project?

**Comment by nlohmann** [comment, marked as answer]
I am not sure what you mean. JSON does not have a version number. This library implements JSON as defined in the latest RFC (https://datatracker.ietf.org/doc/html/rfc8259).

**Reply by AugF** [reply]
Sorry for my poor english.  The "json" version I want to mean is just your project version.  I don't want to contain some syntax of C++14 in my project by using your project.

**Reply by gregmarr** [reply]
You can use all versions in a C++11 project.

**Reply by nlohmann** [reply]
The project targets C++11.

**Reply by AugF** [reply]
Thank you very much.
